nimbus/src/client/mod.rs

```diff
@@ -0,0 +1,40 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+mod fs_client;
+mod http_client;
+use crate::error::{Error, Result};
+use crate::Experiment;
+use fs_client::FileSystemClient;
+use http_client::Client;
+use url::Url;
+
+use crate::ExperimentConfig;
```

Comment:
Naming nit: the name "experiment config" seems pretty misleading, it seems like it should be "remote settings config" or similar.